qa/rpc-tests/wallet_nullifiers.py

```diff
@@ -170,5 +170,46 @@ def run_test (self):
         assert_equal(self.nodes[1].z_getbalance(myzaddr), zaddrremaining2)
         assert_equal(self.nodes[2].z_getbalance(myzaddr), zaddrremaining2)
 
+        # Test viewing keys
+
+        node3mined = Decimal('250.0')
+        assert_equal({k: Decimal(v) for k, v in self.nodes[3].z_gettotalbalance().items()}, {
+            'transparent': node3mined,
+            'private': zsendmany2notevalue,
+            'total': node3mined + zsendmany2notevalue,
+        })
+
+        # add node 1 address and node 2 viewing key to node 3
+        myzvkey = self.nodes[2].z_exportviewingkey(myzaddr)
+        self.nodes[3].importaddress(mytaddr1)
+        self.nodes[3].z_importviewingkey(myzvkey)
+
+        # Check the address has been imported
+        assert_equal(myzaddr in self.nodes[3].z_listaddresses(), False)
+        assert_equal(myzaddr in self.nodes[3].z_listaddresses(True), False)
```

Comment:
This is why I didn't spot @arielgabizon's issue - I copy-pasted this test line, and forgot to change the comparison result to `True`, so it was successfully testing that the method failed :sweat_smile: